src/main/kotlin/no/nav/syfo/client/ereg/EregOrganisasjonResponse.kt

```diff
@@ -8,3 +8,13 @@ data class EregOrganisasjonNavn(
 data class EregOrganisasjonResponse(
     val navn: EregOrganisasjonNavn,
 )
+
+fun EregOrganisasjonResponse.virksomhetsnavn(): String {
+    return this.navn.let {
+        if (it.redigertnavn?.isNotEmpty() == true) {
```

Comment:
== true er redundant (?)

Reply:
Nei, faktisk ikke. If-blokken tåler ikke en verdi som  er kanskje-boolean(Boolean?). Da klager kompilatoren.

Reply:
Men jeg ser at et alternativ er: `if (!it.redigertnavn.isNullOrBlank()) {`

Reply:
Går an å destructure this.navn og endre til return if også. Synes det blir mer lesbart uten .let og it, men litt smak og behag
`val (navnelinje1, redigertnavn) = this.navn;
return if (redigertnavn.isNullOrBlank()) navnelinje1 else redigertnavn`

Reply:
Blir litt det samme for min del, men kan prøve det du foreslår her for å kjenne litt på det.